docs/src/operate/c8y/software-management.md

```diff
@@ -4,101 +4,172 @@ tags: [Operate, Installation, Software Management]
-distributions, which use the **apt** package manager(Ex: RaspberryPi OS , Ubuntu, Debian), from Cumulocity cloud.
+## Management
+
+%%te%% enables you to manage the software installed on your device. The following sections details how to install and remove software packages.
+
+:::info
+For more detailed information on the Cumulocity IoT **Device Management** features, please check out the [official documentation](https://cumulocity.com/docs/device-management-application/managing-device-data/#managing-software).
 :::
 
-Below steps show how to download, install and enable %%te%% software management feature.
+### To add a new software or software version {#add-to-repo}
 
-## Download and install software management packages on the device
+1. Navigate to the *Management* &rarr; *Software repository* page
 
-As a prerequisite, install [tedge and tedge-mapper](../../install/index.md) if not installed already.
+    ![Select software](images/software-repo-1-list.png)
 
-The %%te%% software management packages are in repository on GitHub: [%%te%%](https://github.com/thin-edge/thin-edge.io/releases).
+    The page will display all of the software packages which are are available for to be installed on a device.
 
-To download the package from github repository use the following command (use desired version):
+2. Select *Add software* and fill in the details
 
-```sh
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/<package>_<version>_<arch>.deb
-```
+    <p align="center">
+        <img width="40%" src={require('./images/software-repo-2-add.png').default} alt="Add software" />
+    </p>
 
-Where:
-- `version` -> %%te%% software management components version in x.x.x format
-- `arch` -> architecture type (amd64, armhf)
+    :::tip
+    * If you are using the `apt` software type and hosting the package is hosted in an external repository, then just use an a single space in the *Profile a file path* option.
+    :::
 
-Download `tedge-apt-plugin` and `tedge-agent`
+3. Select *Add software* to create the item
 
-```sh title="Example"
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.9.0/tedge-apt-plugin_0.9.0_armhf.deb
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.9.0/tedge-agent_0.9.0_armhf.deb
-```
+### To install software on a device {#install}
 
-Once the packages are downloaded, proceed to installation.
+1. Select the software you want to install on the device
 
-To install `tedge-apt-plugin` and `tedge-agent` on a %%te%% device do:
+    ![Select software](images/software-install-1-select.png)
 
-```sh
-sudo dpkg -i tedge-apt-plugin_<version>_<arch>.deb
-sudo dpkg -i tedge-agent_<version>_<arch>.deb
-```
+2. Review the software changes
 
-:::note
-Software management feature will be enabled after installation if the device
-is connected to the Cumulocity cloud using:
-```sh
-sudo tedge connect c8y
-```
-:::
+    ![Select software](images/software-install-1-select.png)
```

Comment:
Wrong image

```suggestion
    ![Select software](images/software-install-2-review.png)
```

Reply:
Good catch...I also updated the image description to suite (both in the install and remove section).

Resolved by 9bbb6ea96be81ae5930fe1882091fc6980200238